debian/postinst

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# postinst script for nss-envhosts
+#
+# see: dh_installdeb(1)
+
+set -e
+
+case "$1" in
+    configure)
+    ;;
+
+    abort-upgrade|abort-remove|abort-deconfigure)
+    ;;
+
+    *)
+        echo "postinst called with unknown argument \`$1'" >&2
+        exit 1
+    ;;
+esac
+
+# dh_installdeb will replace this with shell code automatically
+# generated by other debhelper scripts.
+
+#DEBHELPER#
+
+ldconfig
+
+# Add entry to nsswitch.conf
+sed -i 's|\(^hosts:.*\)files|\1envhosts files|g' /etc/nsswitch.conf
```

Comment:
This is sketching me out a little, particularly since we _also_ overwrite this line entirely in `robot-common-cfg` (though this package and that one should hopefully never be installed on the same machine...)

See some related discussion for other packages (in the Red Hat world) where there is uncertainty as to how to modify this file in a responsible way:

https://github.com/authselect/authselect/issues/77#issuecomment-443185158
https://github.com/authselect/authselect/issues/51

If nothing else, please:

- Make this idempotent— if the line already contains `envhosts`, it should go unchanged.
- Make it remove `envhosts` upon uninstallation.

Reply:
I went with the latter approach. Made it idempotent.